src/desktop/apps/article/components/layouts/Article.tsx

```diff
@@ -1,33 +1,49 @@
-import ArticleModel from "desktop/models/article.coffee"
-import InfiniteScrollArticle from "../InfiniteScrollArticle"
-import { once } from "lodash"
-import PropTypes from "prop-types"
 import React from "react"
+import { once } from "lodash"
+
 import { Article } from "reaction/Components/Publishing"
-import Cookies from "desktop/components/cookies/index.coffee"
-import _SuperArticleView from "desktop/components/article/client/super_article.coffee"
-import { setupFollows, setupFollowButtons } from "../FollowButton.js"
-import mediator from "desktop/lib/mediator.coffee"
+import { ArticleProps } from "reaction/Components/Publishing/Article"
+import {
+  ModalOptions,
+  ModalType,
+} from "@artsy/reaction/dist/Components/Authentication/Types"
+
+import InfiniteScrollArticle from "../InfiniteScrollArticle"
+import { setupFollows, setupFollowButtons } from "../FollowButton"
+
+const _SuperArticleView = require("desktop/components/article/client/super_article.coffee")
+const ArticleModel = require("desktop/models/article.coffee")
+const Cookies = require("desktop/components/cookies/index.coffee")
+const mediator = require("desktop/lib/mediator.coffee")
```

Comment:
I wasn't sure how Coffeescript files should be imported but the transpiler doesn't seem to like a normal `import` so I used `require`.Open to suggestions.

Reply:
I think this is a standard way of handling this kind of thing